### ISSUE TITLE
support for pandoc grid-tables

### ISSUE THREAD

**Issue by mangecoeur** [burodepeper/language-markdown#155]
It would be nice to also have support for pandoc grid tables. The table syntax gets applied in the 'data' rows but the divider rows starting with `+=` or `+-` aren't recognized. The main advantage of grid-tables is when the cell contents spans several lines. 

```
+---------------+---------------+--------------------+
| Fruit         | Price         | Advantages         |
+===============+===============+====================+
| Bananas       | $1.34         | - built-in wrapper |
|               |               | - bright color     |
+---------------+---------------+--------------------+
| Oranges       | $2.10         | - cures scurvy     |
|               |               | - tasty            |
+---------------+---------------+--------------------+
```

**Comment by burodepeper** [comment]
I'll have a look.

**Comment by burodepeper** [comment]
Just merged #156 into `master`. Should be part of the next release.

**Comment by mangecoeur** [comment]
👍
